backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlSchema.java

```diff
@@ -395,6 +395,27 @@ private static void migrateTransaction(
     }
   }
 
+  private static void checkRefback(TableMetadata mergeTable, List<TableMetadata> mergeTableList) {
+    for (Column column : mergeTable.getColumns()) {
+      if (!column.isReferenceWithRefback()) continue;
+
+      Column refbackColumn = column.getReferenceRefback();
```

Comment:
where do you check for refbacks in existing tables that are not being dropped (might even be in other schema)? I would expect that to happen in SqlTableMetadata

Reply:
Changed it under the assumption that if it there is refback and its not changed within this migration, it is still there and an exception is thrown

Reply:
Also added a test to see if a refback exist on another schema an error still gets triggered